Alignment/CommonAlignment/python/tools/trackselectionRefitting.py

```diff
@@ -208,13 +225,24 @@ def getSequence(process, collection,
     ####################
 
     if saveCPU:
+        if cosmicTrackSplitting:
+            raise ValueError("Can't turn on both saveCPU and cosmicTrackSplitting at the same time")
         mods = [("TrackSelector", "Alignment", {"method": "load"}),
                 ("TrackRefitter", "First", {"method": "load",
                                             "clone": True}),
                 ("TrackHitFilter", "Tracker", {"method": "load"}),
                 ("TrackFitter", "HitFilteredTracks", {"method": "import"})]
         options["TrackSelector"]["Alignment"].update(
             options["TrackSelector"]["HighPurity"])
+    elif cosmicTrackSplitting:
+        mods = [("TrackRefitter", "First", {"method": "load",
+                                            "clone": True}),
+                #("TrackHitFilter", "Tracker", {"method": "load"}),
```

Comment:
I was trying to be consistent as much as possible with the old version.  I think I also tested and it did weird things when I applied the hit filter, but I don't remember the details.